rundeckapp/grails-app/views/menu/home.gsp

```diff
@@ -59,7 +59,7 @@
     <!-- VUE CSS MODULES -->
     <asset:stylesheet href="static/css/components/version-notification.css"/>
     <!-- /VUE CSS MODULES -->
-
+    <asset:javascript src="static/pages/login.js"/>
```

Comment:
I had to add this, I'm not sure why SVG files would be black and white without this.